src/main/java/net/minecraftforge/client/gui/overlay/VanillaGuiOverlay.java

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.client.gui.overlay;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import net.minecraft.client.Minecraft;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.effect.MobEffects;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.level.GameType;
+import net.minecraft.world.scores.Objective;
+import net.minecraft.world.scores.PlayerTeam;
+import net.minecraft.world.scores.Scoreboard;
+import org.jetbrains.annotations.NotNull;
+import org.lwjgl.opengl.GL11;
+
+/**
+ * All the vanilla {@link IGuiOverlay HUD overlays} in the order that they render.
```

Comment:
The label should be in plain text, so the tag should be `{@linkplain}`.
```suggestion
 * All the vanilla {@linkplain IGuiOverlay HUD overlays} in the order that they render.
```